frontend/src/components/ContractForm.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+
+const WEEKLY = 'Weekly';
+const MONTHLY = 'Monthly';
+
+type PayoutFrequency = WEEKLY | MONTHLY;
+
+type Contract = {
+  recipientAddress: string;
+  salary: number;
+  payoutFrequency: PayoutFrequency;
+};
+
+export const ContractForm = () => {
+  const { register, handleSubmit, errors } = useForm<Contract>();
+
+  const Select = (): JSX.Element => {
+    const [payoutFrequency, setPayoutFrequency] = useState<PayoutFrequency>(WEEKLY);
+
+    return (
+      <select
+        value={payoutFrequency}
+        onBlur={(ev: React.ChangeEvent<HTMLSelectElement>): void =>
+          setPayoutFrequency(ev.target.value)
+        }
+      >
+        <option value="{WEEKLY}">{WEEKLY}</option>
+        <option value="{MONTHLY}">{MONTHLY}</option>
```

Comment:
the contract is currently hard-coded to just monthly, do you intend to just use weekly as a mock first? or are we going for an entire feature? if latter, we need to make changes to the contract for a variable payout